cg/meta/upload/scoutapi.py

```diff
@@ -64,6 +73,7 @@ def build_samples(self, analysis_obj: models.Analysis, hk_version_id: int = None
                 "sex": link_obj.sample.sex,
                 "bam_path": bam_path,
                 "mt_bam": mt_bam_path,
+                "tissue_type": lims_sample.get("source") or "unknown",
```

Comment:
```suggestion
                "tissue_type": lims_sample.get("source", "unknown"),
```
Shorthand version, not sure it is more clear though

Reply:
It is, thanks!